frontend/src/constants/links.ts

```diff
@@ -0,0 +1,4 @@
+// honestly there's no nice place for this in our folder structure and I really, really need this to change in one place
+// might be deletable once we're done with the initial feedback phase
+export const RELEASE_TEMPLATE_FEEDBACK =
+    'https://docs.google.com/forms/d/1ElbScYxbAhFcjQWgRinifoymYHeuXzqIoQXfpUVYGR8/preview';
```

Comment:
```suggestion
    'https://docs.google.com/forms/d/e/1FAIpQLSfbbqsz6wQSgTuJ0NO6ooykTGC_osrzB5-qMrQkbda9SRwnxg/viewform?usp=preview';
```
Just checked, there's a responder link we should use this one

Reply:
Thank you!